conf/messages.cy

```diff
@@ -28,7 +28,8 @@ app.common.no                                               = Na
 ## Capture Company Number
 capture-company-number.title                                = Beth yw rhif cwmni’r bartneriaeth?
 capture-company-number.heading                              = Beth yw rhif cwmni’r bartneriaeth?
-capture-company-number.line_1                               = Gallwch <a id="companiesHouse" href={0} target="_blank"> chwilio yn Nhŷ’r Cwmnïau am rif cofrestru’r cwmni (yn agor tab newydd)</a>.
+capture-company-number.line_1.part1                         = Gallwch
+capture-company-number.line_1.part2                         = chwilio yn Nhŷ’r Cwmnïau am rif cofrestru’r cwmni (yn agor tab newydd)
```

Comment:
I think you might need to remove "(yn agor tab newydd)" as I think this is added by the new component.